<p>You must not use the GO TO instruction in new developments</p>
<h2>Noncompliant Code Example</h2>
<pre>
GO TO 
</pre>
<h2>Compliant Solution</h2>
<pre>
Debe eliminar los GO TO del codigo.

&copy; Carlos Mella
</pre>
