<p>No se debe usar de instrucción HANDLE CONDITION e IGNORE CONDITION en programas CICS. </p>
<h2>Noncompliant Code Example</h2>
<pre>
HANDLE CONDITION e IGNORE CONDITION 
</pre>
<h2>Compliant Solution</h2>
<pre>
Debe eliminar los HANDLE CONDITION e IGNORE CONDITION del codigo.

&copy; Carlos Mella
</pre>
